config/routes.rb

```diff
@@ -244,6 +244,9 @@
     get 'terms' => 'terms#index'
   end
 
+  # Relay
+  get ".well-known/social-relay" => "social_relay#well_known"
```

Comment:
Where does it say to use `x-` prefix? I read it. I'm going to submit the well-known for registration :)